### ISSUE TITLE
it give it text using "text" attribute but it's not showing anything

### ISSUE THREAD

**Issue by Devendra0110** [nstudio/nativescript-floatingactionbutton#111]

**Comment by bradmartin** [comment]
I don't believe `text` has ever been supported with this plugin, due to the native classes not providing that OOTB. It is possible to achieve this but would require an enhancement to the source. Would be happy to help with any PRs to make this happen.

However, if you're on a time crunch, making circular buttons in NS is simple with the `border-radius` styling option and from there you could put text inside the standard NS button.

**Comment by Devendra0110** [comment]
Well, it's given in the [API](https://github.com/nstudio/nativescript-floatingactionbutton#api).

> Allows to use text instead of image

**Comment by bradmartin** [comment]
Oh wow yeah you're right. Guess I've never used it 😁. Then seems like a bug or usage issue. Does that icon work in a simple Label? If so then we can confirm you have the font loaded and the CSS for the font correct.

**Comment by Devendra0110** [comment]
Yeah it's working perfectly with Label but not with FAB

**Comment by dosomder** [comment]
It does work for me. I use `text.decode="&#xe5db;"` and Nativescript-vue

**Comment by DimitrisRK** [comment]
> Yeah it's working perfectly with Label but not with FAB

You forgot to add required 'fas' or 'far' class which is needed for FontAwesome icons.
